swagger.yaml

```diff
@@ -583,6 +583,30 @@ paths:
           $ref: '#/responses/InstanceNotFound'
         500:
           $ref: '#/responses/InternalError'
+  /instances/{instance_id}/dimensions/{dimension}:
+  put:
```

Comment:
The indentation is wrong here. You need to add 2 more spaces. You can use http://editor.swagger.io/#/ to test out the swagger spec